guide/src/python_from_rust.md

```diff
@@ -479,5 +479,22 @@ class House(object):
 }
 ```
 
+## Handling system signals/interrupts (ctrl_c)
+To ensure, that system signals are caught in Rust, you should disable them in Python.
+```rust
+Python::with_gil(|py| {
+            py.run(
+                r#"
+import signal
+print("[python] Ignoring system signals")
+signal.signal(signal.SIGINT, signal.SIG_DFL)
+                "#,
+                None,
+                None,
+            )
+            .unwrap();
+        });
```

Comment:
Python::with_gil(|py| {
    let signal = py.import("signal")?;
    // Set SIGINT to have the default action
    signal
        .getattr("signal")?
        .call1((signal.getattr("SIGINT")?, signal.getattr("SIG_DFL")?))?;
  });